### ISSUE TITLE
What is the release schedule for alertmanager?

### ISSUE THREAD

**Issue by majolo** [prometheus/alertmanager#2761]
We require this feature: https://github.com/prometheus/common/commit/4bfa954c24f5ae9811682883069d53cbf21053c8 which has been merged in Prometheus/common but not yet released in Alertmanager.

Does Alertmanager have a predictable release cadence?

Thanks,
Majolo

**Comment by roidelapluie** [comment]
Hello, it's not predictable but we will cut a release next week.

**Comment by majolo** [comment]
Great thanks @roidelapluie !

**Comment by roidelapluie** [comment]
This is in main but documentation needs updating.

**Comment by majolo** [comment]
Hey @roidelapluie I noticed this commit https://github.com/prometheus/common/commit/2af6d036253eee1a9a08c6ddf6be6d67537bcdff is not part of the 0.32.0 release of common. Is this necessary for the feature (TLS config for Oauth2) to work?

**Comment by freeseacher** [comment]
yep. i waiting for opsgenie improvements too. i\m thinking of using version from master branch. is it stable enough ?

**Comment by majolo** [comment]
Hey @roidelapluie is there an update on this, it would be very useful for us! Happy to help whichever way I can.

**Comment by simonpasquier** [comment]
v0.24.0 should be available once https://github.com/prometheus/alertmanager/pull/2862 merges.
